src/pages/starter-kit/checkout/connect.md

```diff
@@ -18,19 +18,27 @@ To use the Adobe Commerce HTTP Client, update `COMMERCE_BASE_URL=<commerce_insta
 
 Depending on your Adobe Commerce setup, there are two options to authenticate and communicate with App Builder:
 
-- [Configure Adobe Identity Management Service (IMS)](#configure-adobe-identity-management-service-ims)
+- [Configure Adobe Identity Management Service (IMS)](#adobe-identity-management-service-ims)
 
 - [Configure Commerce Integration](#create-a-commerce-integration)
 
-If a Commerce integration is detected, it has precedence over IMS authentication. However, if neither option is detected or configured, than client instantiation will fail.
+If a Commerce integration is detected, it has precedence over IMS authentication. However, if neither option is detected or configured, then client instantiation will fail.
 
-### Configure Adobe Identity Management Service (IMS)
+### Adobe Identity Management Service (IMS)
 
-IMS authentication requires the following setup:
+<InlineAlert variant="info" slots="text1, text2"/>
 
-1. Configure IMS for Commerce following the steps in [Configure the Commerce Admin Integration with Adobe ID](https://experienceleague.adobe.com/en/docs/commerce-admin/start/admin/ims/adobe-ims-config).
+&#8203;<Edition name="paas" /> This process requires a Commerce instance with [Adobe Identity Management Service (IMS) for Adobe Commerce](https://experienceleague.adobe.com/docs/commerce-admin/start/admin/ims/adobe-ims-integration-overview.html) configured.
 
-1. Create new IMS credentials through the [Adobe Developer Console](https://developer.adobe.com/console). Add a new **API** service in the workspace. From the list of APIs, select **I/O Management API** and follow the displayed steps. After adding the API, all credentials are generated.
+SaaS instances already include IMS configuration.
+
+Use the following steps to create OAuth credentials for App Builder authentication:
+
+1. Access your IMS credentials through the [Adobe Developer Console](https://developer.adobe.com/console). Select the project and workspace you set up during the [initial configuration](./getting-started.md#initial-configuration). Then click **OAuth Server-to-Server** in the side-navigation menu.
+
+   <InlineAlert variant="info" slots="text"/>
+
+   The OAuth Server-to-Server option only displays if you have the Adobe I/O Management API enabled, which is done as part of the [initial configuration](./getting-started.md#initial-configuration).
```

Comment:
I think this inline alert is not wrapping the content below, could you check it once more?

Something very picky about wording, maybe `added` sounds more natural than `enabled` (since it’s not something we turn on or off but add to the Console project).
```suggestion
   The OAuth Server-to-Server option displays if you have added the Adobe I/O Management API, which is done as part of the [initial configuration](./getting-started.md#initial-configuration).
```

Everything else looks good to me other than that; please feel free to go ahead 😊